### PR TITLE
Questionnaire items

### PR DESCRIPTION
**Pull request by janagoetze**
- I adjusted the answer scales to fit the questions and moved two questions so that questions of the same form appear together
- I also removed the numbers from the scale because they do not relate to the questions/answers
- some minor reformulations or additions

@kontogiorgos this i a suggestion. feel free to comment so I can go back and adjust what we don't want or dislike. anyways I think we're almost done, looks very nice already 🥳

### CODE REVIEW

File: study1/study1.html [hunk 1]
```diff
@@ -130,114 +131,111 @@ <h2>Questionnaire</h2>
...
-			<label for="info3">&nbsp;3<br/>
+			<label for="info3"><br/>
 				<input type="radio" name="info" value=3 id="info3"/>
 			</label>&nbsp;&nbsp;&nbsp;
-			<label for="info3">too much information</label>
+			<label for="info3">Agree</label>
 			<br><br>
 
 			<!-- Collaboration -->
 			<p>Mathew's instruction was collaborative</p>
```
Comment (janagoetze) [inline study1/study1.html:187]: remove this one

Comment (janagoetze) [reply study1/study1.html:187]: move to final questionnaire

Comment (janagoetze) [reply study1/study1.html:187]: replace by "machine-like" vs "human-like"

**Review by kontogiorgos — COMMENTED** [review]
changing shortly on other branch.

**Pull request merged**